bazel/glog.bzl

```diff
@@ -92,6 +92,9 @@ def glog_library(namespace = "google", with_gflags = 1, **kwargs):
     darwin_only_copts = [
         # For stacktrace.
         "-DHAVE_DLADDR",
+        # Avoid deprecated syscall().
+        "-DHAVE_PTHREAD_THREADID_NP",
+        "-Werror",
```

Comment:
I'm not sure it's a good idea to add the flag here and for macOS only. @drigz Is there a way to enable `-Werror` exclusively for CI builds?